landoapi/secapproval.py

```diff
@@ -0,0 +1,83 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Functions related to the Security Bug Approval Process.
+
+See https://wiki.mozilla.org/Security/Bug_Approval_Process.
+"""
+import inspect
+
+from landoapi.projects import get_sec_approval_project_phid
+
+dedent = inspect.cleandoc
+
+
+def send_sanitized_commit_message_for_review(revision_phid, message, phabclient):
+    """Send a sanitized commit message for review by the sec-approval team.
+
+    See https://wiki.mozilla.org/Security/Bug_Approval_Process.
+
+    Args:
+        revision_phid: The PHID of the revision to edit.
+        message: The sanitized commit message string we want to be reviewed.
+        phabclient: A PhabClient instance.
+    """
+    comment = format_sanitized_message_comment_for_review(message)
+    sec_approval_phid = get_sec_approval_project_phid(phabclient)
+    phabclient.call_conduit(
+        "differential.revision.edit",
```

Comment:
This should return a list of transactions that you applied back to you. I thought the plan was to save the phid in the database, to make sure we're using the correct one? It'd also make the searching for the comment much quicker in your followup PRs.

Reply:
Changed.